PublishParasoftResults/BuildApiClient.ts

```diff
@@ -169,8 +171,17 @@ export class BuildAPIClient {
             method: "GET",
             headers: headers
         };
-        const response = await fetch(downloadUrl, options);
-
-        return response.arrayBuffer();
+        try {
+            const response = await fetch(downloadUrl, options);
+            if (response.status === undefined || response.status < 200 || response.status >= 300) {
+                tl.warning(`Download artifact error: ${downloadUrl}, error code: ${response.status}`);
```

Comment:
Let's update the error message according to the following example:

An error(xxx) occurred while attempting to download artifact from: xxx.